Filelist

```diff
@@ -834,15 +833,24 @@ RT_SCRIPTS =	\
 		runtime/syntax/generator/update_date.vim \
 		runtime/syntax/generator/vim.vim.base \
 
-# Unix runtime
+# Unix-like runtime.
 RT_UNIX =	\
 		README_unix.txt \
+		runtime/doc/*.awk \
+		runtime/doc/Makefile \
+		runtime/doc/vim.1 \
+		runtime/doc/evim.1 \
+		runtime/doc/vimdiff.1 \
+		runtime/doc/vimtutor.1 \
+		runtime/doc/xxd.1 \
+		runtime/indent/Makefile \
+		runtime/pack/dist/opt/editorconfig/mkzip.sh \
```

Comment:
I don't think those manpages belong to $RT_UNIX

Reply:
Well, I assumed that they apply to all Unix-like systems. Although, of course, I may be wrong. But anyway, Windows doesn't have a built-in tool for viewing manpages.